Subnet-EVM config table (#1932)
ava-labs/avalanche-docs: content/docs/nodes/chain-configs/subnet-evm.mdx

|---------------------|--------|-----------------------------|---------|
| `local-txs-enabled` | `bool` | Enables local transactions. | `false` |

### Transaction Pool Settings

| Option                     | Type      | Description                                                        | Default                                             |
|----------------------------|-----------|--------------------------------------------------------------------|-----------------------------------------------------|
| `tx-pool-price-limit`      | `uint64`  | Minimum gas price (in wei) for the transaction pool.               | From `legacypool.DefaultConfig.PriceLimit`          |
| `tx-pool-price-bump`       | `uint64`  | Minimum price bump percentage to replace an existing transaction.  | From `legacypool.DefaultConfig.PriceBump`           |
| `tx-pool-account-slots`    | `uint64`  | Max executable transaction slots per account.                      | From `legacypool.DefaultConfig.AccountSlots`        |
| `tx-pool-global-slots`     | `uint64`  | Max executable transaction slots for all accounts.                 | From `legacypool.DefaultConfig.GlobalSlots`         |

> | `tx-pool-account-queue`    | `uint64`  | Max non-executable transaction slots per account.                  | From `legacypool.DefaultConfig.AccountQueue`        |

Reviewer: ```suggestion
| `tx-pool-account-queue`    | `uint64`  | Max non-executable transaction slots per account.                  | 64    |
```

> | `tx-pool-global-queue`     | `uint64`  | Max non-executable transaction slots for all accounts.             | From `legacypool.DefaultConfig.GlobalQueue`         |

Reviewer: ```suggestion
| `tx-pool-global-queue`     | `uint64`  | Max non-executable transaction slots for all accounts.             | 1024    |
```

> | `tx-pool-lifetime`         | `Duration`| Maximum time a transaction can remain in the pool.                 | From `legacypool.DefaultConfig.Lifetime`            |

Reviewer: ```suggestion
| `tx-pool-lifetime`         | `Duration`| Maximum time a transaction can remain in the pool.                 |  10 Minutes     |
```

| `offline-pruning-data-directory`| `string` | Data directory for offline pruning.              | `""` (empty) |

## VM2VM Network

| Option                         | Type    | Description                                | Default |
|--------------------------------|---------|--------------------------------------------|---------|
| `max-outbound-active-requests` | `int64` | Max number of outbound active requests.    | `16`    |

## Sync Settings

| Option                       | Type      | Description                                                      | Default                   |
|------------------------------|-----------|------------------------------------------------------------------|---------------------------|
| `state-sync-enabled`         | `bool`    | Enables state synchronization.                                   | `false`                   |
| `state-sync-skip-resume`     | `bool`    | Forces state sync to use highest available summary block.        | `false`                   |
| `state-sync-server-trie-cache` | `int`   | Cache size for state sync server trie in MB.                     | `64`                      |

> | `state-sync-ids`             | `string`  | IDs for state sync.                                              | `""` (empty)              |

Reviewer: ```suggestion
| `state-sync-ids`             | `string`  | Node IDs for state sync.                                              | `""` (empty)              |
```

| `skip-upgrade-check`       | `bool`            | Disables checking that upgrades occur before last accepted block. | `false`      |
| `accepted-cache-size`      | `int`             | Depth to keep in the accepted headers and logs cache.      | `32`               |
| `transaction-history`      | `uint64`          | Max blocks from head whose transaction indices are reserved. | `0` (no limit)   |
| `tx-lookup-limit`          | `uint64`          | **Deprecated**, use `transaction-history` instead.         |                    |
| `skip-tx-indexing`         | `bool`            | Skips indexing transactions; useful for non-indexing nodes.| `false`            |
| `warp-off-chain-messages`  | `[]hexutil.Bytes` | Encoded off-chain messages to sign.                        | `[]` (empty list)  |

## RPC Settings

| Option            | Type     | Description                        | Default        |
|-------------------|----------|------------------------------------|----------------|
| `http-body-limit` | `uint64` | Limit for HTTP request body size.  | Not specified  |

> ## Database Configuration

Reviewer: ```suggestion
## Standalone Database Configuration
```

| Option                     | Type      | Description                               | Default        |
|----------------------------|-----------|-------------------------------------------|----------------|

> | `use-standalone-database`  | `*PBool`  | Use a standalone database.                | `false`        |

Reviewer: ```suggestion
| `use-standalone-database`  | `*PBool`  | Use a standalone database. By default Subnet-EVM uses a standalone database if no block was accepted.                 | `nil`        |
```
